schemas/activity-stream/impression-stats/impression-stats.1.schema.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
This file is auto-generated. See the new file under `templates` for the user-created definition.